### DISCUSSION TITLE
Why were User Tags removed?

### DISCUSSION THREAD

**Discussion by dragow0lf** [xgi/houdoku#161] [Q&A]
They were useful in situations where you wanted to exclude a particular tag from results or only have one tag come up in results.

**Comment by xgi** [comment]
Internally the feature was error-prone with how it associated tags with series. There wasn't a straightforward solution so I opted to remove them.

I do plan on adding "Categories" that you can assign series to and will be usable in a similar way. I may also add support for filtering by regular tags with the main library search field.
